WordPress/WordPressShare/BlogPickerViewController.swift

```diff
@@ -0,0 +1,135 @@
+import Foundation
+import UIKit
+import WordPressShared
+import WordPressComKit
+
+
+class BlogPickerViewController : UITableViewController
```

Comment:
SitePickerViewController instead of BlogPickerViewController we need to move away from blog :)